### PR TITLE
libaarhusxyz PR 29 - Flightline splitter

### PR DESCRIPTION
**Pull request by burningsage**

### CODE REVIEW

**Comment by burningsage** [comment]
closes https://github.com/emerald-geomodelling/libaarhusxyz/issues/30

**Comment by burningsage** [comment]
Also, this issue https://github.com/emerald-geomodelling/libaarhusxyz/issues/31 is resolved by this pull request

File: libaarhusxyz/gex.py [hunk 1]
```diff
@@ -171,8 +171,8 @@ def gate_times(self, channel='Channel1'):
             
         return gate_time_array[remove_gates_from:remove_gates_from+no_gates,:] + gex[channel].get('GateTimeShift', 0.0) + gex[channel].get('MeaTimeDelay', 0.0)
 
-    def __getattr__(self, name):
-        return self.gex_dict[name]
```
Comment (redhog) [inline libaarhusxyz/gex.py:175]: Why did you remove this?

File: libaarhusxyz/xyz.py [hunk 2]
```diff
@@ -458,10 +459,11 @@ def plot(self, fig = None):
         for line_no, ax in zip(lines, axs):
             self.plot_line(line_no, ax)
 
-    def calculate_xdist(self):
-        dxdist = np.insert((  (self.flightlines[self.x_column].iloc[1:].values - self.flightlines[self.x_column].iloc[:-1].values)**2
-                            + (self.flightlines[self.y_column].iloc[1:].values - self.flightlines[self.y_column].iloc[:-1].values)**2)**0.5, 0,0)
-        self.flightlines["xdist"] = np.cumsum(dxdist)
+    # FIXME: Which function for 'calculate_xdist'?!? the one on 152 or 451?!?
```
Comment (redhog) [inline libaarhusxyz/xyz.py:462]: Why did you remove this?

**Comment by redhog** [comment]
Superseeded by changes copied manually to master
